docker-compose.yml

```diff
@@ -5,7 +5,13 @@ services:
     container_name: foodloop-postgres
     volumes:
       - ./foodloop-data:/var/lib/postgresql/data
-      - ./foodloop-db-files/dump.sql:/docker-entrypoint-initdb.d/dump.sql
+      - ./foodloop-db-files/schema.sql:/docker-entrypoint-initdb.d/schema.sql
+      - ./foodloop-db-files/insertFood.sql:/docker-entrypoint-initdb.d/insertFood.sql
+      - ./foodloop-db-files/insertFoodToTag.sql:/docker-entrypoint-initdb.d/insertFoodToTag.sql
+      - ./foodloop-db-files/insertRestaurant.sql:/docker-entrypoint-initdb.d/insertRestaurant.sql
+      - ./foodloop-db-files/insertRestaurantToFood.sql:/docker-entrypoint-initdb.d/insertRestaurantToFood.sql
+      - ./foodloop-db-files/insertRestaurantToTag.sql:/docker-entrypoint-initdb.d/insertRestaurantToTag.sql
+      - ./foodloop-db-files/insertTag.sql:/docker-entrypoint-initdb.d/insertTag.sql
```

Comment:
maybe we can further compartmentalize these files into a `seeders` folder, this way it would be easy to execute/not execute that specific folder depending on whether we need it

Reply:
okay, done!